es-ds-components/.eslintrc.json

```diff
@@ -0,0 +1,133 @@
+{
+    "parser": "vue-eslint-parser",
+    "parserOptions": {
+        "parser": "@typescript-eslint/parser"
+    },
+    "settings": {
+        "import/resolver": {
+            "typescript": {
+                "alwaysTryTypes": true
+            },
+            "alias": {
+                "map": [
+                    ["@", "./lib"],
+                    ["~", "."]
+                ]
+            }
+        }
+    },
+    "plugins": ["@typescript-eslint"],
```

Comment:
I wonder if something like `plugin:vue/vue3(-strongly)?-recommended` might be worth using, maybe: